interface_config.js

```diff
@@ -49,9 +49,10 @@ var interfaceConfig = {
     TOOLBAR_BUTTONS: [
         'microphone', 'camera', 'closedcaptions', 'desktop', 'fullscreen',
         'fodeviceselection', 'hangup', 'profile', 'info', 'chat', 'recording',
-        'livestreaming', 'etherpad', 'sharedvideo', 'settings', 'raisehand',
-        'videoquality', 'filmstrip', 'invite', 'feedback', 'stats', 'shortcuts',
-        'tileview', 'videobackgroundblur', 'download', 'help'
+        'livestreaming', 'etherpad', 'presenter', 'sharedvideo', 'settings',
+        'raisehand', 'videoquality', 'filmstrip', 'invite', 'feedback', 'stats',
+        'shortcuts', 'tileview', 'videobackgroundblur', 'download', 'help',
+        'videocropforeground'
```

Comment:
If a new toolbar button configuration is needed, remember to follow up with any important consumers of the iframe api and to look at branding to make sure it has the right configuration.